components/tag-list/tag-list-item-mixin.js

```diff
@@ -72,7 +67,7 @@ export const TagListItemMixin = superclass => class extends superclass {
 		/** @ignore */
 		this.role = 'listitem';
 		/** @ignore */
-		this.tabindex = -1;
```

Comment:
Nit: I'm not totally sure, but probably don't still need the `@ignore` on this as I don't _believe_ the docs care about non-properties? Maybe I'm wrong though.